### DISCUSSION TITLE
颜色失效了，大家又遇到吗

### DISCUSSION THREAD

**Discussion by wjxyy** [izhangzhihao/intellij-rainbow-brackets#2665] [Q&A] [answered]
![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/106721609/9fcbbe59-1f64-4c6c-97ca-3e0ccb1ff3c6)

**Comment by wjxyy** [comment]
![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/106721609/7f278d2e-36a4-4a57-a099-c93929366c24)
有的类可以有的就失效了

**Comment by izhangzhihao** [comment]
文件长度超过 1000 行了吗？

**Reply by wjxyy** [reply]
超过了大佬

**Comment by izhangzhihao** [comment, marked as answer]
那就取消这个勾选或者设置一个更大的值：
<img width="943" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/60b58e97-b72b-4ecf-a501-89728309ed18">

**Reply by wjxyy** [reply]
好的 谢谢了已解决了
